<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>线程</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnStartThread">
          <property name="text">
           <string>启动线程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiceBegin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiceEnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStopThread">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>结束线程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>清空文本</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="LabPic">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../../../../../resource/Qt/Source/QT5.12Samp2019/chap13Thread/samp13_1ThreadSignal/res.qrc">:/dice/images/d0.jpg</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LabA">
        <property name="text">
         <string>Thread状态：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../../resource/Qt/Source/QT5.12Samp2019/chap13Thread/samp13_1ThreadSignal/res.qrc"/>
 </resources>
 <connections/>
</ui>
